project.json

```
@@ -1,4 +1,4 @@
 {
   "name": "surrealdb.js",
-  "version": "0.9.0"
+  "version": "0.9.1"
```

Would like to do this in a separate commit :)
```suggestion
  "version": "0.9.0"
```